macdown-cmd/main.m

```diff
@@ -47,6 +81,16 @@ int main(int argc, const char * argv[])
             [argproc printHelp:YES];
         else if (argproc.printsVersion)
             [argproc printVersion:YES];
+        
+        NSData *dataFromPipe = MPPipedData();
+        
+        if (dataFromPipe) {
+            // Store piped content in a temporary file which will be read by MacDown on launch
+            NSString *fileName = [NSString stringWithFormat:@"%@_%@", [[NSProcessInfo processInfo] globallyUniqueString], @"pipedText.txt"];
+            NSURL *fileURL = [NSURL fileURLWithPath:[NSTemporaryDirectory() stringByAppendingPathComponent:fileName]];
+            [[NSFileManager defaultManager] createFileAtPath:fileURL.path contents:dataFromPipe attributes:nil];
```

Comment:
If we don’t care about attributes, this can probably just use `[dataFromPipe writeToURL:fileURL options:0 error:&error]`. This has the advantage of being able to report errors.